### PR TITLE
Adding germany to the files

### PR DESCRIPTION
**Pull request by ssheytanov**
I wanted to add germany data

### CODE REVIEW

File: countries.json
Comment (sheytanov) [inline countries.json:1]: I don't like the structure of this file. Please add countries array and add the countries inside of it.

File: germany.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        "CHE"
+      ],
+      "nativeName": "Deutschland",
+      "numericCode": "276",
+      "flags": {
+        "svg": "https://flagcdn.com/de.svg",
+        "png": "https://flagcdn.com/w320/de.png"
+      },
+      "currencies": [
+        {
```
Comment (sheytanov) [inline germany.json:49]: What happened to their old currency?

**Review by sheytanov — CHANGES_REQUESTED** [review]
Please check my comments and change the corresponding files

File: countries.json
Comment (ssheytanov) [reply countries.json:1]: Resolved

File: germany.json (hunk 1 above)
Comment (ssheytanov) [reply germany.json:49]: They don't use it

**Review by sheytanov — APPROVED** [review]

**Pull request merged**